src/demo-app/a11y/list/list-a11y.html

```diff
@@ -0,0 +1,71 @@
+<div class="demo-list">
+  <section>
+    <h2> Seasoning </h2>
+    <p>Showing a non-interactive list of seasonings.</p>
+    <md-list>
+      <md-list-item *ngFor="let item of items"> {{item}} </md-list-item>
+    </md-list>
+  </section>
+
+
+  <section>
+    <h2>Mailbox navigation</h2>
+    <p>Showing a navigation list with links to google search, and "more information" icon button</p>
+    <md-nav-list>
```

Comment:
- Made another component `MdNavList` for `mat-nav-list` with `role="navigation"`
- Removed `role="list"` for `MdList` 
- Added "accessibility" section in list.md (Same as grid-list)